code/modules/mob/living/carbon/xenomorph/castes/defiler/abilities_defiler.dm

```diff
@@ -13,6 +13,7 @@
 
 /datum/action/xeno_action/activable/defile/on_cooldown_finish()
 	playsound(owner.loc, 'sound/voice/alien_drool1.ogg', 50, 1)
+	owner.balloon_alert(owner, "Defile ready")
```

Comment:
Do we really want to add balloon alerts for cooldown finish? This is already visually and audio respresented so this feels more like clutter